docs/content/usage/packages/conan.en-us.md

```diff
@@ -52,17 +51,19 @@ Publish a Conan package by running the following command:
 conan upload --remote={remote} {recipe}
 ```
 
-| Parameter | Description |
-| ----------| ----------- |
-| `remote`  | The remote name. |
-| `recipe`  | The recipe to upload. |
+| Parameter  | Description           |
+| ---------- | --------------------- |
+| `remote` | The remote name.      |
+| `recipe` | The recipe to upload. |
 
 For example:
 
 ```shell
 conan upload --remote=gitea ConanPackage/1.2@gitea/final
 ```
 
+You cannot publish a package if a package of the same name and version already exists. You must delete the existing package first.
```

Comment:
That's not correct because Conan works on a files. So the "package file twice" sentense would be correct here.